src/main/java/emissary/test/core/ExtractionTest.java

```diff
@@ -290,7 +305,20 @@ protected void checkStringValue(Element meta, String data, String tname) {
         String key = meta.getChildTextTrim("name");
         String value = meta.getChildText("value");
         String matchMode = "equals";
+        String validateField = "false";
```

Comment:
Would it be better to have this as a boolean? 
Then when the vf.getValue() is called it can convert the value to boolean state.